capstone/src/main/java/com/google/univiz/api/MapData.java

```diff
@@ -0,0 +1 @@
+package com.google.univiz.api;
```

Comment:
i'm actually not sure if it's ok to have an empty `.java` file. @ccondit do you know? Other than that i'm ok with the package organization. Note that since you've named this file `MapData.java` technically you have already picked the interface name. So you might as well add it here:

```java
/** */
public interface MapData {}
```

Reply:
a noted! will do!

Reply:
i think it would be fine, in theory. but i think adding the interface is fine in this PR.